packages/@aws-cdk/example-construct-library/test/integ.example-resource.ts

```diff
@@ -0,0 +1,25 @@
+/*
+ * Our integration tests act as snapshot tests to make sure the rendered template is stable.
+ * If any changes to the result are required,
+ * you need to perform an actual CloudFormation deployment of this application,
+ * and, if it is successful, a new snapshot will be written out.
+ *
+ * For more information on CDK integ tests,
+ * see the main CONTRIBUTING.md file.
+ */
+
+import * as core from '@aws-cdk/core';
```

Comment:
Can we add the stack verification steps block - https://github.com/aws/aws-cdk/blob/master/CONTRIBUTING.md#integration-tests ?

Reply:
What would you put here in this case...? The stack is just a bunch of CFN wait handles...